### PR TITLE
Feat/assessement

### PR DESCRIPTION
**Pull request by abdel-ke**

### CODE REVIEW

File: src/models/User.ts [hunk 1]
```diff
@@ -57,4 +61,16 @@ User.init(
   }
 );
 
+User.hasMany(Assessment, {
+  sourceKey: 'id',
+  foreignKey: 'userId',
+  as: 'assessments'
+});
+
+Assessment.belongsTo(User, {
```
Comment (ouarrtaha) [inline src/models/User.ts:70]: move Assessment.belongsTo to Assessment model

File: src/models/User.ts [hunk 2]
```diff
@@ -57,4 +61,16 @@ User.init(
   }
 );
 
+User.hasMany(Assessment, {
+  sourceKey: 'id',
```
Comment (ouarrtaha) [inline src/models/User.ts:65]: no need of sourceKey and targetKey, as it takes 'id' by default

**Review by ouarrtaha — DISMISSED** [review]

File: src/routes/v1/assessment.route.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+router
+  .route('/')
+  .get(
+    // auth(PERMISSIONS.MANAGE_USERS),
+    assessmentController.getAssessment
+  )
+  .post(
+    // auth(PERMISSIONS.MANAGE_USERS),
+    // validate(userValidation.createUser),
+    assessmentController.postAssessment
```
[the remark this answers is not in the document]
Comment (soof05) [reply src/routes/v1/assessment.route.ts:17]: I attached it to expert audit too

File: src/models/User.ts (hunk 1 above)
Comment (soof05) [reply src/models/User.ts:70]: I was not able to separate Assessement.belongsTo from this file,it keeps giving me an error,
the docs of Sequalize say : **Again, as mentioned, the main way to do it used a pair of Sequelize associations (hasMany and belongsTo).**

**Review by ouarrtaha — APPROVED** [review]

**Pull request merged**